test/unit_tests/braket/default_simulator/test_unitary_matrix_simulation.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import numpy as np
+import pytest
+
+from braket.default_simulator import gate_operations
+from braket.default_simulator.unitary_matrix_simulation import UnitaryMatrixSimulation
+
+evolve_testdata = [
```

Comment:
I wrote these tests based on the tests for the unitary simulator in Qiskit Terra: https://github.com/Qiskit/qiskit-terra/blob/main/test/python/basicaer/test_unitary_simulator.py

Reply:
This should have a copyright statement of Qiskit somewhere in the file as this is obvious a derivative work of the original.